### PR TITLE
Dashboard design first version & filter for book

### PR DESCRIPTION
**Pull request by Thomas905**

### CODE REVIEW

**Review by alexandregrue — DISMISSED** [review]
Ok for me, we develop the fonctionnality together, test look ok to filter on one or more elements and obtained the expected result

File: public/assets/js/dashboard.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+function myFunction() {
```
Comment (RomainClair) [inline public/assets/js/dashboard.js]: You should use more meaningful names

File: src/Model/FormProcessing.php [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        $errors = [];
+        if (!empty($_GET)) {
+            $items = [
+                'author_id' => $_GET['author_id'],
+                'editor_id' => $_GET['editor_id'],
+                'category_id' => $_GET['category_id'],
+                'format_id' => $_GET['format_id'],
+                'location_id' => $_GET['location_id'],
+                'status_id' => $_GET['status_id'],
+            ];
```
Comment (RomainClair) [inline src/Model/FormProcessing.php]: You should verify the presence (and emptiness eventually)  of each $_GET field individually

**Review by RomainClair — CHANGES_REQUESTED** [review]
.DS_Store must not be in the repository

**Review by alexandregrue — APPROVED** [review]

**Pull request closed without merging**